_meeting-notes/2019-10-29.md

```diff
@@ -0,0 +1,143 @@
+  - continuation of discussion from last week, but with Raul
+  - eg. "seeds/peers" combinations https://github.com/ipfs/testground/pull/82
+    - Answer: 
+      - Test Case is a just template/behaviour
+      - The "looping over test variants" sits a layer above the Plans. Examples:
+        - In the Github Integration
+        - In the Test Ground runner process
+      - A Test Case is run with specific parameters on a trigger event. This trigger event can be: CLI call, a Github event, etc
+      - We might get concept of recipes (a base config to run the multiple cases in a plan)
+- (@daviddias, 5min) Trello doesn't seem to be working, we still love Kanban.
+  - People are not visiting Trello often
+  - Information in Trello is different from Github. Requires constant sync'ing
+  - Proposal: Move to ZenHub
+    - Information will all live in Github issues (where developers love it)
+    - It will give the Project Manager the view it needs to understand progress
+    - Open Source Contributors will always have access to the information about the project
+    - Less one spot to maintain information
+  - Votes
+    - Yes: David, Raul
+    - No:
+    - No opinion: Jim
+- (@jimpick, 5min) 
+  - Cluster setup demo (AWS/Ansible)
+- (@jimpick, 5min)
+  - Simplified cluster runner proposal
+  - https://trello.com/c/2MJwZ71P/68-multi-machine-ssh-or-http-api-runner
+
+## :dart: Next
+
+### OKRs for the Quarter
+
+![](https://raw.githubusercontent.com/ipfs/testground/master/docs/img/Q4OKRs.png)
+
+### Top 3 things to get done this week
+
+- "A Test Plan" is complete (end to end, meaning: we spawn nodes, we run the test on AWS infra from our own local box and we see results in Kibana)
+- Prepare for Hack Week (folks, free your schedule, it will be a week of HACKIIING)
+- (When Test Plan 2 is done) Onboard other folks to contribute specific Test Plans to Test Ground
+
+### Assignments
+
+- ==@raulk==
+  - Create an issue to conceptualize the notion of Test Plan Recipes
+  - Preparing the logistics and activities for the Hack Week
+  - Complete the runner local:go
+  - Review the Roadmap PR https://github.com/ipfs/testground/pull/75
+- ==@jimpick==
+  - Get smlbench2 code-reviewed and merged to master somewhere as an example (it's a subset of test plan 2 / bitswap test)
+  - Get ansible automation and packer template merged
+  - Prototype super-simple test runner (non-Docker Swarm) so we can experiment as a team next week
+  - Help Dirk with bitswap test case
+  - Discuss ideas for higher level of automation (auto-provisioning EC2 machines), self-serve platform
+- ==@daviddias==
+  - (Now that Raul is back) Schedule a Pair Programming session to review Test Plan 2 for the Week of Oct 28 (this week)
+  - Deliver Plan 1 - Chew Large Datasets in time for Hack Week
+  - Finish reading the golang programming book 😅
+  - Clarify on what are the expectations on how to use the team Kanban
+  - Move the Kanban to Zenhub
+- ==@stebalien==
+  - ???
```

Comment:
@Stebalien what's your plan for this week?

Reply:
Try to find some time to write test plans.